drivers/mdio/Kconfig

```
@@ -27,6 +27,7 @@ config MDIO_SHELL
 source "drivers/mdio/Kconfig.esp32"
 source "drivers/mdio/Kconfig.sam"
 source "drivers/mdio/Kconfig.nxp_s32_netc"
+source "drivers/mdio/Kconfig.nxp_imx_netc"
```

Nit: keep alphabetical order

```suggestion
source "drivers/mdio/Kconfig.nxp_imx_netc"
source "drivers/mdio/Kconfig.nxp_s32_netc"
```

---

Thanks @pdgendt 
Will change it. And sorry, because I was seeing these exsited were not in order.